protocols/gooddollar/index.json

```diff
@@ -0,0 +1,10 @@
+{
+  "cname": "gooddollar",
+  "description": "GoodDollar - Universal basic income protocol, for the people by the people",
+  "path": "gooddollar",
+  "isEnabled": true,
+  "type": "compoundish",
+  "discourseForum.url": "https://discourse.gooddollar.org",
+  "discourseForum.categoryId": "5",
+  "safeAddress": "0x5C16960F2Eeba27b7de4F1F6e84E616C1977e070"
```

Comment:
Hey, can you change this to treasuryAddress: "0x5C16960F2Eeba27b7de4F1F6e84E616C1977e070"?

Reply:
@Rashmi-278 done

Reply:
Hey, sorry there was a typo on my end, can you please change it to 

`  "treasuryAddresses": ["0x5C16960F2Eeba27b7de4F1F6e84E616C1977e070"]
`

Reply:
Also can you submit this form if you haven't done so ? https://docs.boardroom.info/adding-your-project/adding-your-project/unlocking-the-hub

Reply:
done